types/ioredis/index.d.ts

```diff
@@ -1400,6 +1400,8 @@ declare namespace IORedis {
     type Ok = 'OK';
 
     interface Cluster extends EventEmitter, Commander, Commands {
+        options: ClusterOptions;
```

Comment:
What do you think about marking these as readonly since they are private members. Has been done already for `interface Pipeline`, but not `interface Redis`.

Reply:
Yes that makes sense, I made the change.